template/.github/workflows/preview.yml

```diff
@@ -0,0 +1,27 @@
+name: Netlify Preview
+on:
```

Comment:
[compass](https://nimblehq.co/compass/development/code-conventions/github-actions/#formatting) => We should add a line break between each root-level block

Reply:
1d366c0 ✅